go.mod

```diff
@@ -83,6 +83,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/patrickmn/go-cache v2.1.0+incompatible
```

Comment:
This was discussed with Sam, I think it's been placed here by `go mod tidy` due to the +incompatible flag

As this was done automatically rather than manually we decided to leave it.